pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -296550,7 +296549,7 @@ self: {
        platforms = [
          "aarch64-darwin" "i686-linux" "x86_64-darwin" "x86_64-linux"
        ];
-     }) {inherit (pkgs) xgboost;};
+     }) {xgboost = null;};
```

Comment:
What's up with this line? How did you regenerate this file?

Reply:
I was surprised as well, but this is just the result of running the `regenerate-hackage-packages.sh` script.

Reply:
According to git bisect, the regression happened in 1d63f89caaf140ac33f1796dc0aaeb20f4ac4e55 which was PR  #167016.

Reply:
Indeed, thanks! Fix at #167985. Can you unstage that line manually in the meantime?

Reply:
Done.